packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -41,7 +41,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
       ),
     },
     transformIgnorePatterns: [
-      '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$',
+      '[/\\\\]node_modules[/\\\\](?!@zept).+\\.(js|jsx|mjs|ts|tsx)$',
```

Comment:
Two issue hier:
- What exactly is the `@zept`, and why is listed here?
- If the `webpack` rule for `tsx?` files covers the `node_modules` folder, those files should not be listed on this ignore list.

Reply:
This is a mistake in that I updated the package for our internal use to also ensure tests processed the package as well. I feel including all packages in the transform would slow tests down too. 
I am voting we close this PR as to niche.